NotesApp/Reusable/CustomAlertView.swift

```diff
@@ -90,30 +92,34 @@ final class CustomAlertView: UIView {
         containerView.addSubview(cancelButton)
         containerView.addSubview(deleteButton)
         
+        setupHierarchy()
+    }
+    
+    private func setupHierarchy() {
         NSLayoutConstraint.activate([
             containerView.centerXAnchor.constraint(equalTo: centerXAnchor),
```

Comment:
we cann func setUpHierarchy where we add subviews. this func should have name related to constraints. setUpConstraints or something like that